ip_country/src/test_dbip_country.rs

```diff
@@ -0,0 +1,22 @@
+
+// GENERATED CODE: REGENERATE, DO NOT MODIFY!
+
+pub fn ipv4_country_data() -> (Vec<u64>, usize) {
+    (
+        vec![
+            0xA690000300801003, 0xAA103C48826A0496, 0x1DF5F4C718016207, 0xD001E04BB7C3053C,
+            0x0128BAA60060F124, 0x804A4ACB80875806, 0x0000000000053437,
+        ],
+        413
+    )
+}
+
+pub fn ipv6_country_data() -> (Vec<u64>, usize) {
+    (
+        vec![
+            0x3000040000400007, 0x00C0001400020000, 0xB400150000000700, 0xA004AF8B024C0E23,
+            0x0000000400BC2051,
+        ],
+        302
+    )
+}
```

Comment:
What the hack is this file for??

It probably should be here. If it necessarily needs to be performed, then the `ip_country` crate should have its own `generated` folder (like in Node) and it should be generated inside there.

Again, if you need it, you should make the extra effort and parameterize the tested code so it place the product, this file in such a directory which would be specified by an argument of the function.